nonhuman-firebase-authentication/readme.md

```diff
@@ -0,0 +1,36 @@
+> **The goal of this guide:** Explain how you can enable automated services to communicate with Firebase
+
+# Two types of automatic services communicate with Firebase
+
+[CircleCi](https://circleci.com/) is a [continous integration](https://en.wikipedia.org/wiki/Continuous_integration) service that tests your code. It will for example check whether there are statements that are not closed: { without an }.
+
+Bots listen to events and perform an action based on the type of event. For example, a bot can listen to whether something was added to a Firebase Collection and can notice a specific user of that change.
+> we currently use a bot to push a message to slack when a client adds a message to a conversation on the interface
+
+## Difference between a bot and circle in communicating with Firebase
+Bots use the firebase secret. Go to the firebaseio url of your database and go to secrets.
```

Comment:
I had one but removed it since the guide will go public and the url won't be very useful for most readers. 
Tried to un-New Atoms it :)